services/wcr/legal_entities.md

```diff
@@ -0,0 +1,62 @@
+
+## Business logic
+
+### Legal entity names
+The legal entity name of an entity is the formally registered name of the entity, if any. The business logic for legal entity names varies according to the business type, tier and the date of the registration.
+- Limited Companies and Limited Liability Partnerships:
+  - Upper-tier: The legal entity name is the registered company name retrieved from Companies House during the registration, e.g. "Applied Waste Solutions Limited". Note that this will not be populated for registrations created before the deployment of the legal entities changes.
+  - Lower-tier: Currently nil, as companies house deteails are not retrieved for lower-tier registrations.
+- Sole Traders:
+  - Upper-tier: The legal entity name is the first name and last name of the key person for the registration, e.g. "Jim Smith". This is always populated.
+  - Lower-tier: Currently nil, as the key person name is not stored for lower-tier sole traders.
+- Other:
+  - Legal entity name is nil in all cases.
+
+### Business names
+The business name of an entity is an informal trading name provided by the user creating the registration and not subject to validation. Again the business logic varies according to the business type, tier and date of the registration:
+- Limited Companies and Limited Liability Partnerships:
+  - Upper-tier: The user may add an optional business name during registration, e.g. "We Are Waste". Prior to the deployment of the legal entities changes this field was mandatory.
+  - Lower-tier: The business name is mandatory, as the companies house name is not stored for lower-tier companies.
+- Sole Traders:
+  - Upper-tier: The user may add an optional business name during registration, e.g. "Jim's Skips". Prior to the deployment of the legal entities changes this field was mandatory.
+  - Lower-tier: The business name is mandatory, as the key person name is not stored for lower-tier sole traders.
+- Other:
+  - Business name is mandatory in all cases.
+
+### Presentation name
+"Presentation name" is the term used within the team for a combination of legal entity name and business name. If only one of legal entity name or business name is populated, the presentation name will consist of that field. If both are populated, the presentation name will comprise the legal entity name plus "trading as" plus the business name.
+Examples:
+
+| Business Type | Registered Company Name | Business Name | Key Person Name | Presentation Name                     |
```

Comment:
This is a possibility that a sole trader could use their name as the business name so Jim Smith trading as Jim Smith, not sure if we want to add that?